### PR TITLE
global: docs, PyPy and Flask>=0.11

### PR DESCRIPTION
**Pull request by jirikuncar**
addresses #27

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -114,9 +114,8 @@
         'Programming Language :: Python :: 2',
         'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.3',
-        'Programming Language :: Python :: 3.4',
         'Programming Language :: Python :: 3.5',
-        'Development Status :: 3 - Alpha',
+        'Programming Language :: Python :: Implementation :: PyPy',
```
Comment (tiborsimko) [inline setup.py]: Should we also add `Programming Language :: Python :: Implementation :: CPython` in addition to `PyPy`?

File: RELEASE-NOTES.rst [hunk 2]
```diff
@@ -19,18 +17,18 @@ What's new
...
 ------------
 
-   $ pip install invenio-assets==1.0.0a5
+   $ pip install invenio-assets==1.0.0b1
 
 Documentation
 -------------
 
-   http://pythonhosted.org/invenio-assets/
+   https://pythonhosted.org/invenio-assets/
```
Comment (tiborsimko) [inline RELEASE-NOTES.rst]: `http` works too, but so does `https`.

File: invenio_assets/__init__.py [hunk 3]
```diff
@@ -24,20 +24,38 @@
 
 """Media assets management for Invenio.
 
-First make sure you have Flask application with Click support (meaning
-Flask 0.11+ or alternatively use the Flask-CLI extension):
+Invenio-Assets helps you to integrate webassets, installation of NPM packages,
```
Comment (tiborsimko) [inline invenio_assets/__init__.py]: "helps you with integration of..., installation of..." OR "helps you to integrate..., to install ..."

Shall we attack English cosmetics like this in this PR, or as part of #28?

File: invenio_assets/__init__.py [hunk 4]
```diff
@@ -24,20 +24,38 @@
...
+and process of collecting static files.
+
+If you want to learn details about packages dependencies follow:
```
Comment (tiborsimko) [inline invenio_assets/__init__.py]: "about package dependencies, please see:"

File: invenio_assets/__init__.py [hunk 5]
```diff
@@ -24,20 +24,38 @@
...
     >>> assets = InvenioAssets(app)
 
+During initalization two Flask extensions ``flask_assets.Environment`` and
+:class:`flask_collect.Collect` are instantiated and configured. Also bundles
+specified in entry point group called `'invenio_assets.bundles'` are
+:ref:`automatically registered <entry-points-loading>`.
+
+Bundles
+-------
+Webassets packages provides class ``Bundle`` for creating collection of files
```
Comment (tiborsimko) [inline invenio_assets/__init__.py]: "package provides"

File: invenio_assets/__init__.py [hunk 6]
```diff
@@ -47,16 +65,72 @@
     >>> assets.env.register('mybundle', Bundle())
     <Bundle ...>
 
+.. _entry-points-loading:
+
+Entry points loading
+~~~~~~~~~~~~~~~~~~~~
+Invenio-Assets will automatically bundles defined by the entry point groups
```
Comment (tiborsimko) [inline invenio_assets/__init__.py]: "will automatically load bundles defined..."

File: RELEASE-NOTES.rst (hunk 2 above)
Comment (jirikuncar) [inline RELEASE-NOTES.rst]: `https` everywhere except http://inveniosoftware.org 😢

File: setup.py (hunk 1 above)
Comment (jirikuncar) [inline setup.py]: done

**Pull request merged**